package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack --config webpack.config.prod.js",
-    "build-dev": "webpack --config webpack.config.dev.js",
     "start": "npm run build && ./leapchat",
-    "test": "node_modules/.bin/mocha  --reporter nyan test/.setup.js test/**/*.test.js"
+    "build-dev": "webpack --config webpack.config.dev.js",
```

Comment:
@Spetastian What do you think of having `build` and `build-prod` rather than `build-dev` and `build`, since 95% of the time we'll be building for dev purposes, and I can occasionally type the longer command right before a deploy?